lib/v2/screens/explore_screens/vote_screens/proposals/viewmodels/proposals_list_event.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:meta/meta.dart';
+
+/// --- EVENTS
+@immutable
+abstract class ProposalsListEvent {
+  const ProposalsListEvent();
+}
```

Comment:
@gguijarro-c-chwy 
As of today I do not know why all these abstract classes have to implement Equatable, I mean that it is very rare event null scenary to compare two events of the same type. (i.e. Load('2')==Load('2')).

This is what is known as the error of only copy and paste without knowing if it is really necessary.

Reply:
I aks this on the Flutter_bloc discord channel so.. this is the answear i got.

> 
> You have custom classes, and what if you call InitialLoad() twice in a row?
> They are two separate events and without Equatable, how would you know if you called the same thing twice? From what I understand, without Equatable, the event objects will be recognized as the same and thus the new state will not reflect.
>

However I am not entirely sure events are compared by default or only if a scenario occurs where if I have to compare events. (like you example in the search input where you only dispatch the last event).

For the sake of uniformity I will put Equatable again, in the future I hope to resolve this question.

Reply:
I see, thanks for researching this further